Include/internal/pycore_object.h

```diff
@@ -14,9 +14,11 @@ extern "C" {
 #include "pycore_pystate.h"       // _PyInterpreterState_GET()
 #include "pycore_runtime.h"       // _PyRuntime
 
+#define _PyObject_IMMORTAL_REFCNT 999999999
```

Comment:
I recommend a comment here so no one gets confused about the word "immortal".  Perhaps something like this:

```suggestion
/* This value provides *effective* immortality, meaning the object should never
    be deallocated (until runtime finalization).  See PEP 683 for more details about
    immortality, as well as a proposed mechanism for proper immortality. */
#define _PyObject_IMMORTAL_REFCNT 999999999
```